package.json

```diff
@@ -7,6 +7,11 @@
     "presets": [
       "@babel/preset-env",
       "@babel/preset-react"
+    ],
+    "plugins": [
+      [
+        "@babel/plugin-proposal-class-properties"
```

Comment:
I want to know what this is for as well.

Reply:
this allows you to use function of the format "function = e => {}" .